src/modules/_[spaceId]/HomePage/HomePage.tsx

```diff
@@ -37,6 +41,35 @@ export default function HomePage({
   const sortedIds = useSortedPostIdsByLatestMessage(allPostIds)
   const order = useSortByConfig(sortedIds)
 
+  const [search, setSearch] = useState('')
+  const postsQuery = getPostQuery.useQueries(order)
+
+  let searchResults: PostData[]
```

Comment:
The code is run each rerender. Should we wrap it into useMemo and add dependency?

Reply:
updated